rollup.config.js

```diff
@@ -1,8 +1,12 @@
 import babel from "@rollup/plugin-babel";
 import { terser } from "rollup-plugin-terser";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
```

Comment:
nodeResolve is needed to be able to bundle node_modules (dom-element-getter-helpers)